pypose/lietensor/lietensor.py

```diff
@@ -735,11 +735,347 @@ def randn(self, *size, sigma=1.0, requires_grad=False, **kwargs):
         data = torch.cat([rotation, scale], dim=-1)
         return LieTensor(data, ltype=rxso3_type).requires_grad_(requires_grad)
 
+class so2Type(LieType):
+    def __init__(self):
+        super().__init__(1, -1, 1)  # dimension, embedding, manifold
```

Comment:
I am unsure what `embedding` refers to. Thus, for now it's set to `-1` for all classes (`so2Type`, `SO2Type`, `se2Type` and `SE2Type`).

Reply:
it refers to the dimension of data, for the representation of that specific class, its lie group, and lie manifold, respectively. The definition is  [here](https://github.com/pypose/pypose/blob/1d2ba781c48706ba0526cce5254af270f1fd7548/pypose/lietensor/lietensor.py#L36).
```python
        self._dimension = torch.Size([dimension]) # Data dimension
        self._embedding = torch.Size([embedding]) # Embedding dimension
        self._manifold  = torch.Size([manifold])  # Manifold dimension
```